.github/workflows/ci.yaml

```diff
@@ -78,13 +100,18 @@ jobs:
           retry_wait_seconds: 30
           command: |
             composer update --optimize-autoloader --no-interaction --no-progress ${{ matrix.composer-flags }}
+            composer info -D | sort
```

Comment:
This is redundant with output of `composer update`.

Reply:
I find `-D` and `sort` useful
actually even more useful than output of `update`. maybe let's use `update -q` ?

Reply:
That would leave us with less information than actually provided.

Reply:
so... maybe let's keep both?

Reply:
let's merge as-is and we can always adjust later ;)